components/community-feed.tsx

```
@@ -387,7 +388,8 @@ export default function CommunityFeed() {
         </div>
       </div>
 
-      <div className="flex flex-col md:flex-row items-start md:items-center justify-between bg-muted/50 p-4 rounded-lg gap-4">
+<div className="flex flex-col md:flex-row items-start md:items-center justify-between bg-muted/50 p-4 rounded-lg gap-4">
+
```

ESLint

[error] Replace `<div·className="flex·flex-col·md:flex-row·items-start·md:items-center·justify-between·bg-muted/50·p-4·rounded-lg·gap-4">⏎` with `······<div·className="flex·flex-col·md:flex-row·items-start·md:items-center·justify-between·bg-muted/50·p-4·rounded-lg·gap-4">`

(prettier/prettier)